City-of-Helsinki/berth-reservations-admin
# VEN-908 | Add "no berth" functionality

## Description :sparkles:

* Add "no berth" functionality
* Add more button options to ButtonWithConfirmation.tsx
* Hide berth accessibility behind disabled feature flag

## Issues :bug:

### Closes :no_good_woman:

* [VEN-908](https://helsinkisolutionoffice.atlassian.net/browse/VEN-908): FE - Implement no-berth option

## Testing :alembic:

### Manual testing :construction_worker_man:

* "No berth" functionality should exist in application list (expanded row), application view and customer view
  * "No berth" button should not be visible if the customer is missing
  * It should be possible to mark unhandled berth applications with "no berth"
  * It should be possible to still handle the application after marking it with "no berth" (waiting for available berth)
